Mathlib/Topology/Algebra/Order/Compact.lean

```diff
@@ -193,6 +193,73 @@ theorem IsCompact.exists_isLUB [ClosedIciTopology α] {s : Set α} (hs : IsCompa
   IsCompact.exists_isGLB (α := αᵒᵈ) hs ne_s
 #align is_compact.exists_is_lub IsCompact.exists_isLUB
 
+theorem _root_.cocompact_le_atBot_atTop [LinearOrder α] [CompactIccSpace α] :
```

Comment:
```suggestion
theorem cocompact_le_atBot_atTop [LinearOrder α] [CompactIccSpace α] :
```
If I'm not mistaken, you're already in the root namespace, so no need to prefix it. Same thing below.